packages/core/admin/components/EditView/Permalink/index.tsx

```diff
@@ -10,15 +10,18 @@ interface Props {
 
 const EditViewRightLinks: React.FC<Props> = ({ path }) => {
   const [url, setUrl] = useState<string>();
+  const fetchClient = useFetchClient();
 
   useEffect(() => {
-    request('/webtools/info/config', { method: 'GET' })
-      .then((res: Config) => {
-        setUrl(res.website_url);
+    fetchClient.get('/webtools/info/config')
+      .then((response) => {
+        const configData = response.data as Config;
```

Comment:
Instead of doing `as Config`, you could add it as a Generic to the get function.
Like so: `fetchClient.get<Config>('/webtools/info/config')`